.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
[OPTIONAL]
- Add directories that aren't from your project to your .gitignore file. (.vscode)